habitat_baselines/rl/ver/report_worker.py

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import contextlib
+import functools
+import os
+import time
+from collections import defaultdict
+from multiprocessing.context import BaseContext
```

Comment:
May also want to extend from torch.multiprocessing if available. It's either a shim to multiprocessing or alternatively has some subclasses which are more optimized for passing around Torch Tensors.

Reply:
All the imports are just to typing. I get multiprocessing context from torch to create everything.

Reply:
@erikwijmans Ah, my bad.